pipeline/scraper/npm_query.py

```diff
@@ -75,7 +79,7 @@ def run_query():
     cur = database.cursor()
 
     # Fetch all package names from the database
-    cur.execute("SELECT name FROM packages;")
+    cur.execute("SELECT name FROM packages WHERE name LIKE 'pkg:npm/%';")
```

Comment:
this may impact the speed, a separate column with a flag or a type would be much faster than partial string match.

Reply:
You're right, but I also just tested this on our newly enlarged dataset and it takes about half a second, so I'm going to say this is good enough for now. We can investigate breaking out type into a separate field later.